views/12-months.html

```diff
@@ -0,0 +1,27 @@
+{% extends "layout.html" %}
+
+{% block pageTitle %}
+  You cannot use this service
+{% endblock %}
+
+{% block content %}
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <h1 class="govuk-heading-xl">We cannot grant an extension to the filing deadline</h1>
+    <p class="govuk-body-m">
+      The last filing date for {{companyName}} ({{companyNumber}})
+      is already 12 months or more after the end of the accounting reference period.
+    </p>
+    <p>
+      Filing extensions are granted to allow a company to meet its filing obligations.
+      They cannot be used to extend the filing deadline to more than 12 months after the end of the accounting reference period.
+    </p>
+    <p>
+      You must
+      <a href="https://www.gov.uk/file-your-company-annual-accounts">file your company accounts</a>
```

Comment:
does this link actually open in a new tab? I thought the default behaviour for an a href would open the link in the current tab?

Reply:
I don't know I have just gone with what is present in the screenshot on the ticket.

Reply:
Cannot really test this until it is merged as it requires the work for LFA-1911 to be done in full.

Reply:
I am reasonably confident that this would not open in a new tab.
Could you add `target="_blank"` to the <a> tag, which should make it open in a new tab

Reply:
updated this.